src/pkg/intents/generated.go

```diff
@@ -0,0 +1,159 @@
+// Code generated by github.com/Khan/genqlient, DO NOT EDIT.
+
```

Comment:
aren't you missing some genqlient configs here? genqlient.yaml, its graphql, and a generate.go file?

Reply:
🤯
You are an awesome reviewer!
Thanks, fixed